sdk/storage/test/datalake/file_system_client_test.cpp

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "file_system_client_test.hpp"
+
+#include <algorithm>
+
+namespace Azure { namespace Storage { namespace Test {
+
+  const size_t c_PATH_TEST_SIZE = 5;
+
+  std::shared_ptr<DataLake::FileSystemClient> FileSystemClientTest::m_fileSystemClient;
+  std::string FileSystemClientTest::m_fileSystemName;
+
+  std::vector<std::string> FileSystemClientTest::m_PathNameSetA;
+  std::vector<std::string> FileSystemClientTest::m_PathNameSetB;
+  std::string FileSystemClientTest::m_DirectoryA;
+  std::string FileSystemClientTest::m_DirectoryB;
+
+  void FileSystemClientTest::SetUpTestSuite()
+  {
+    m_fileSystemName = LowercaseRandomString(10);
```

Comment:
The default value is `10`, so what's the pointe of specifying it explicitly?

Reply:
Removed. However, the caller should not be aware of the default value. If someone changes the default value to other values, the 10 will work.
It is not a big deal in test code.